seatunnel-engine/seatunnel-engine-storage/imap-storage-file/src/main/java/org/apache/seatunnel/engine/imap/storage/file/IMapFileStorage.java

```diff
@@ -221,7 +223,7 @@ public Set<Object> loadAllKeys() {
     }
 
     @Override
-    public void destroy() {
+    public void destroy(boolean deleteAllFileFlag) {
```

Comment:
why do you create a new method?

Reply:
> why do you create a new method?

Sometimes I don't need to delete files, is there a better solution? Just like jdbc closes the connection but keeps the data.